tests/pytests/CMakeLists.txt

```diff
@@ -37,13 +33,15 @@ foreach(n ${PY_TEST_FILES})
 		COMMAND bash -c "MODARGS='UNION_ITERATOR_HEAP 1;' ${baseCommand} -t ${n}"
 		WORKING_DIRECTORY ${PROJECT_BINARY_DIR})
 
-    add_test(NAME "PY_${test_name}_RAW_DOCID_ENCODING"
-		COMMAND bash -c "MODARGS='RAW_DOCID_ENCODING true;' ${baseCommand} -t ${n}"
-		WORKING_DIRECTORY ${PROJECT_BINARY_DIR})
+	if (NOT USE_COVERAGE)
+		add_test(NAME "PY_${test_name}_RAW_DOCID_ENCODING"
+			COMMAND bash -c "MODARGS='RAW_DOCID_ENCODING true;' ${baseCommand} -t ${n}"
+			WORKING_DIRECTORY ${PROJECT_BINARY_DIR})
+	endif()
```

Comment:
No need for coverage of `RAW_DOCID_ENCODING`?

Reply:
Not sure it worth the time spent there. Ariel recommended we remove it.